tracer/src/Datadog.Trace.BenchmarkDotNet/DatadogExporter.cs

```diff
@@ -63,7 +63,7 @@ public IEnumerable<string> ExportToFiles(Summary summary, ILogger consoleLogger)
 
                 foreach (var report in summary.Reports)
                 {
-                    Span span = tracer.StartSpan("benchmarkdotnet.test", startTime: startTime);
+                    Span span = tracer.StartSpanWithTags("benchmarkdotnet.test", startTime: startTime);
```

Comment:
Is this the right name if we're _not_ passing tags?

Reply:
true it feels weird that tags can be null with that naming

Reply:
I agree, I would prefer to change `Tracer.StartActiveWithTags()` => `Tracer.StartActive()`, and leave `Tracer.StartSpan()` as-is. That said, you run into overload issues then, so it's easier said than done. Maybe just leaving `StartSpan` as-is for now is best?

These are internal APIs though, so I'm inclined to not bikeshed it too much, as we can easily change them later.

Reply:
I'm thinking of renaming this to `StartSpanInternal` and `StartActiveInternal`. The reason I'd like to add the `Internal` suffix is that should make it a lot easier for me to audit the code and make sure that can identify and de-risk NullReferenceException questions from this PR. What do you think of that change?

Reply:
I made the above changes in 3e9abec783745ef390f17b9ca770dbfa488a1508 and f2018129427bfdd33a0df4f4ef600c12539cffa7 . Let me know if what you think